### ISSUE TITLE
Can this product will work on Angular SSR ?

### ISSUE THREAD

**Issue by OmkarJadhav21** [creativetimofficial/ct-material-dashboard-pro-angular#285]
I am trying to convert this project for the ssr but it will gives me the lots of error due to the used JQuery in project.May I know is their any other solution for this or Can I get same product in Angular SSR ( Server Side Rendering ) ?

**Comment by chelaruc** [comment]
Hi, @OmkarJadhav21! Thank you for using our product and sorry for the late response. Have you found a solution for this? Best, Ciprian

**Comment by OmkarJadhav21** [comment]
No,I am still working on that but I think we can solve this by identifing the platforms i.e. Server or browser but it makes some complications.
